modules/common/tools/Makefile

```diff
@@ -62,3 +89,12 @@ ifndef CLIENT_GEN_BINARY
 else
 	@echo "[tools] client-gen already exists"
 endif
+
+.PHONY: install-yarn
+install-yarn:
+ifndef YARN_BINARY
+	@echo "[tools] downloading yarn..."
+	@npm install yarn
```

Comment:
I'm not sure if we should force install like that. I'm using brew to install most things, for sure others have their preference too. Yarn is well-known tool, I'd just check if it's installed and if not print some feedback message.

Reply:
Agree. We shouldn't install this for the user. We also do not install Go for them. Tools installation is only for Go dependencies and we make sure that Go is installed first.